api/src/controllers/StatementController.js

```diff
@@ -16,14 +16,16 @@ const aggregate = (req) => {
   const maxTimeMS = Number(req.query.maxTimeMS) || MAX_TIME_MS;
   const maxScan = Number(req.query.maxScan) || MAX_SCAN;
   const pipeline = JSON.parse(req.query.pipeline);
+  const sampleSize = Number(req.query.sampleSize) || -1;
```

Comment:
I think I would prefer to default this to undefined since we don't usually allow error-like values like this in Learning Locker's code.

Reply:
OK, changed it to `undefined`.

Reply:
Sweet thank you! 👍